src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -288,4 +288,44 @@ struct DerivCompute
       tmpl::list<VariablesTag, domain::Tags::Mesh<Dim>, InverseJacobianTag>;
 };
 
+/*!
+ * \ingroup DataBoxTagsGroup
+ * \brief Computes the spatial derivative of a single tensor tag not in a
+ * Variables.
+ *
+ * Computes the spatial derivative for a single tensor represented by
+ * 'TensorTag' in the frame mapped to by 'InverseJacobianTag'. It takes a
+ * single Tensor designated by 'TensorTag', the inverse Jacobian, and a mesh
```

Comment:
Sorry, I don't understand how the mesh is optional. It's hard-coded and so looks to be required

Reply:
oh hmm I may not entirely understand how our compute tags work then cause I've been using Tags::DerivTensorCompute<TensorTag, InverseJacobian> and since that worked I guess I just assumed that it's not required for the mesh to be passed in to work.

Reply:
Everything in the `arguments_tags` is what is "passed in" The template parameters are just used to have one compute tag template that can be adjusted for different purposes. Template here means the same as in the colloquial sense, it's a general template that you can specialize to specific purposes.

Reply:
Oh I see, that makes more sense, thank you!